packages/errors/src/errors.ts

```diff
@@ -1491,6 +1491,25 @@ export const AllCypressErrors = {
       `
   },
 
+  MIGRATION_MISMATCHED_CYPRESS_VERSIONS: (version: string) => {
+    return errTemplate`
+      You are running Cypress version 10 in global mode, but you are attempting to migrate a project where ${fmt.cypressVersion(version)} is installed. 
+
+      Ensure the project you are migrating has Cypress version 10 installed.
+
+      https://on.cypress.io/migration-guide
+    `
+  },
+
+  MIGRATION_CYPRESS_NOT_FOUND: () => {
+    return errTemplate`
+      You are running Cypress version 10 in global mode, but we were unable to determine the version of Cypress the project you are attempting to migrate is using.
+
+      Ensure the project you are migrating has Cypress version 10 installed.
+
+      https://on.cypress.io/migration-guide
```

Comment:
I felt like this wasn't super clear, I think because there are only 2 cases where we are "unable to determine the version":

1. if the `resolve-from` fails for some reason, which would be a bug
2. if `cypress/package.json` DOES exist but `version` is empty, which also seems like it could only be a bug

Since `!version` is always either a bug or a sign the project definitely doesn't have `cypress@10`, we should make the error message more direct:

```suggestion
      You are running Cypress 10 in global mode and attempting to migrate a project where an install of `cypress` cannot be found.

      Ensure that `cypress@10` is installed in the project you are attempting to migrate.

      https://on.cypress.io/migration-guide
```